example/pubspec.lock

```diff
@@ -73,6 +73,7 @@ packages:
       path: ".."
       relative: true
     source: path
+    version: "0.0.3"
```

Comment:
Can you check what version is in your pubspec.yaml please? When I check out your branch I have issues running `flutter pub get`. I think yours still says 0.0.3.

Reply:
Fixed it

Reply:
Ok, I'll check it out when I'm back online later